api/hearings/hearing.mock.ts

```diff
@@ -190,17 +147,6 @@ export const init = () => {
   });
 
   mock.onPost(linkedHearingGroup).reply(config => {
-    // START : This few lines code jus to faciliate testing for specific hearing id of h100014
-    // so that even the failure scenarios can be verified
-    const jsonData = JSON.parse(config.data);
-    const hearingId = jsonData.hearingsInGroup[0].hearingId
-    if (hearingId === 'h100014') {
-      return [
-        500,
-        null,
-      ];
-    }
-    // END
     return [
```

Comment:
any reason to remove all those unhappy path?

Reply:
> 
@rayliangatos - @Vamshi-Muniganti requested earlier to remove if possible when he has done the testing but haven't had a chance before so doing right now, is it ok ?

Reply:
@SivaprakasamSivaprakasam as this mock, no much impact on the happy path, but it can be refered to unhappy path, I would prefer to keep this in the mock response.